support-frontend/assets/pages/supporter-plus-landing/formSections/amountAndBenefits.tsx

```diff
@@ -27,6 +33,14 @@ export function AmountAndBenefits({
 	addBackgroundToBenefitsList?: boolean;
 	isCompactBenefitsList?: boolean;
 }): JSX.Element {
+	const { internationalisation } = useContributionsSelector(
+		(state) => state.common,
+	);
+	const campaignSettings = getCampaignSettings('Us_eoy_2023');
+
+	const showUSCampaignTicker =
```

Comment:
I've probably missed it - but where's the switch to make sure this isn't enabled yet?

Reply:
It's `window.guardian.enableContributionsCampaign` or `window.guardian.settings.switches.campaignSwitches.enableContributionsCampaign`. It's driven from the RRCP the same as our other switches for things like payment methods.

Currently we only have an on/off switch for 'enable any campaign' as I think it's only ever been used for one thing at a time in the past. Improving this in the future is something George and I were discussing earlier today.